sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -989,6 +989,10 @@ public partial class BlobBaseClient
         public virtual Azure.Storage.Blobs.Specialized.BlobBaseClient WithSnapshot(string snapshot) { throw null; }
         protected virtual Azure.Storage.Blobs.Specialized.BlobBaseClient WithSnapshotCore(string snapshot) { throw null; }
     }
+    public static partial class BlobClientExtensions
```

Comment:
This should go in [SpecializedBlobExtensions](https://apiview.dev/Assemblies/Review/a17b6175b93847db83a99c7fcfeca9ab#Azure.Storage.Blobs.Specialized.SpecializedBlobExtensions) so we don't add a new type.

Reply:
I'll mirror this in Queues as well. There needed to be a new type no matter what, but now it will be in line with this Blobs change.